### ISSUE TITLE
Getting unknown property: verbb\formie\fields\formfields\Phone::countryLabel

### ISSUE THREAD

**Issue by Timusan** [verbb/formie#279]
**Description**

The Phone field keeps throwing errors since the latest Formie update when it is set as required, either:

`Getting unknown property: verbb\formie\fields\formfields\Phone::countryLabel` or `Getting unknown property: verbb\formie\fields\formfields\Phone::numberLabel` when the country drop down is disabled. All other fields seem to work just fine.

**Steps to reproduce**
1. Create a new form
2. Add a phone field and make it required
3. Add to template (via `{{ craft.formie.renderForm('formHandle') }}`)
4. Submit

### Form settings

- Multi-page form: No
- Submission Method: Ajax or Page Reload (does not matter)
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.7
- Craft version: 3.5.17.1
- Multi-site: No

**Comment by Timusan** [comment]
Never mind ... this seemed to be caused by using a third-party script to track form submissions within Google Analytics which prevents the form submission, sends the GA data and then uses the default JS API's `form.submit()` function to submit the form instead of using Formies submission handler and submitting via `submitHandler.submitForm()`.

I am curious though as why this crashes the submission process on the Phone field, only when it is required, and not on other fields?

**Comment by engram-design** [comment]
Hmmm, not quite sure off the top of my head, so I'll see what I can do to reproduce this. Is there any custom JS code on your end worth sharing? Otherwise, I can test by just preventing submission with JS.

**Comment by Timusan** [comment]
Not really, there is no other custom JS hooking in to this form at the moment.

**Comment by Timusan** [comment]
Hmm, after some more testing it seems to fail even with the correct form submit handler as well. This is a simple custom script I just hacked together which triggers this error ... as far as I know this is the correct way of going about intervening in a Formie submission process, correct?

``` js
if(document.getElementById('myForm')) {
    let form = document.querySelector('#myForm form');
    let submitHandler = false;

    form.addEventListener('onBeforeFormieSubmit', (e) => {
        submitHandler = e.detail.submitHandler;
    });

    form.addEventListener('onFormieValidate', (e) => {
        e.preventDefault();

        if(submitHandler) {
            environment === 'live' ? ga('send', 'event', 'my_form', 'submit') : null;
            submitHandler.submitForm();
        }
    });
}
```

**Comment by Timusan** [comment]
Aha, it is caused by this line:

`environment === 'live' ? ga('send', 'event', 'my_form', 'submit') : null;`

It only occurs when the site is not in `live` mode and it trips on the `null` being inserted there. Not sure why it would choke on that though? Rewriting it a bit less sloppy makes it work in all site modes:

```js
if(environment === 'live')
  ga('send', 'event', 'my_form', 'submit')
```

So I am guessing this is more of a problem between chair and keyboard, but still interested into why it makes the submission crash.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.8](https://github.com/verbb/formie/releases/tag/1.3.8).
